### ISSUE TITLE
Update Peer dependencies for React 19

### ISSUE THREAD

**Issue by ahmadi3d** [iamhosseindhv/notistack#614]
Hello
Can you please update the Peer Dependencies for React 19 ? I believe none of the changes are breaking changes, Thank you!

**Comment by viktor21minds** [comment]
any progress on this?

**Comment by RunRanger** [comment]
Since there're PRs open over 2 month ago i believe this repository is dead...

**Comment by BarishSarac** [comment]
https://github.com/iamhosseindhv/notistack/issues/615

**Comment by ahmadi3d** [comment]
@BarishSarac The code you sent does it support stacking and the enter exit animation too ?

**Comment by RunRanger** [comment]
I would recommend to switch to 
[https://fkhadra.github.io/react-toastify/introduction/](react-toastify)
or 
([https://sonner.emilkowal.ski/](sonner)

**Comment by ahmadi3d** [comment]
@RunRanger Yes, I found React-Toastify to be the best alternative as well. However, considering the refactoring involved, I’ll likely write my own wrapper and create a structure similar to Notistack. Inside the wrapper, I’ll use React-Toastify. This way, if I ever want to change the library in the future, I won’t have to refactor everything again.

**Comment by Zitrooone** [comment]
+1 for this, even though it seems unlikely that something will happen.

**Comment by Ben-CA** [comment]
+1 for supporting React 19 without --legacy-peer-deps

**Comment by nagy135** [comment]
the crazy part here is that big ui library like MUI depends on this dead library to not render toasts on top of each other

**Comment by Ben-CA** [comment]
Yeah, with 3M downloads a month, and 836 dependencies on NPM, it would be nice if we could just get React 19 support added...

**Comment by nagy135** [comment]
thank you @iamhosseindhv !

**Comment by iamhosseindhv** [comment]
React 19 is now listed as a peer dep in the latest release `v3.0.2`. Sorry about the delay

**Comment by Ben-CA** [comment]
Thank you for updating @iamhosseindhv !
